ts/store/reducers/backendStatus.ts

```diff
@@ -182,7 +182,7 @@ export const isCGNEnabledSelector = createSelector(
 );
 
 /**
- * return the remote config about CGN enabled/disabled
+ * return the remote config about FIMS enabled/disabled
  * if there is no data, false is the default value -> (CGN disabled)
```

Comment:
```suggestion
 * if there is no data, false is the default value -> (FIMS disabled)
```
This is a typo, isn't it?